packages/components/src/components/Tabs/__tests__/Tabs.ct.tsx

```diff
@@ -0,0 +1,516 @@
+        await expect(dropdownContent.getByText(THIRD_TAB_TRIGGER_TEXT)).toBeVisible();
+    });
+
+    test('should allow selecting tabs in dropdown', async ({ mount, page }) => {
+        const FIRST_TAB_TRIGGER_TEXT = 'First Tab with long test after';
+        const SECOND_TAB_TRIGGER_TEXT = 'Second Tab with long test after';
+        const THIRD_TAB_TRIGGER_TEXT = 'Third Tab with long test after';
+        const component = await mount(
+            <Tabs.Root data-test-id={TABS_ROOT_TEST_ID}>
+                <Tabs.Tab value="first">
+                    <Tabs.Trigger data-test-id={FIRST_TAB_TRIGGER_TEST_ID}>{FIRST_TAB_TRIGGER_TEXT}</Tabs.Trigger>
+                    <Tabs.Content data-test-id={FIRST_TAB_CONTENT_TEST_ID}>First Content</Tabs.Content>
+                </Tabs.Tab>
+                <Tabs.Tab value="second" disabled>
+                    <Tabs.Trigger data-test-id={SECOND_TAB_TRIGGER_TEST_ID}>{SECOND_TAB_TRIGGER_TEXT}</Tabs.Trigger>
+                    <Tabs.Content data-test-id={SECOND_TAB_CONTENT_TEST_ID}>Second Content</Tabs.Content>
+                </Tabs.Tab>
+                <Tabs.Tab value="third">
+                    <Tabs.Trigger data-test-id={THIRD_TAB_TRIGGER_TEST_ID}>{THIRD_TAB_TRIGGER_TEXT}</Tabs.Trigger>
+                    <Tabs.Content data-test-id={THIRD_TAB_CONTENT_TEST_ID}>Third Content</Tabs.Content>
+                </Tabs.Tab>
+            </Tabs.Root>,
+        );
+
+        await expect(component.getByTestId(FIRST_TAB_TRIGGER_TEST_ID)).toBeInViewport();
+        await expect(component.getByTestId(SECOND_TAB_TRIGGER_TEST_ID)).toBeInViewport();
+        await expect(component.getByTestId(THIRD_TAB_TRIGGER_TEST_ID)).not.toBeInViewport();
+        await expect(component.getByTestId(FIRST_TAB_TRIGGER_TEST_ID)).toHaveAttribute('data-state', 'active');
+        await expect(component.getByTestId(SECOND_TAB_TRIGGER_TEST_ID)).not.toHaveAttribute('data-state', 'active');
+        await expect(component.getByTestId(THIRD_TAB_TRIGGER_TEST_ID)).not.toHaveAttribute('data-state', 'active');
+        await expect(component.getByTestId(FIRST_TAB_CONTENT_TEST_ID)).toBeVisible();
+        await expect(component.getByTestId(SECOND_TAB_CONTENT_TEST_ID)).not.toBeVisible();
+        await expect(component.getByTestId(THIRD_TAB_CONTENT_TEST_ID)).not.toBeVisible();
+
+        await component.getByTestId(DROPDOWN_TRIGGER_TEST_ID).click();
+        const dropdownContent = page.getByTestId(DROPDOWN_CONTENT_TEST_ID);
+        await expect(dropdownContent).toBeVisible();
+        await expect(dropdownContent.getByText(SECOND_TAB_TRIGGER_TEXT)).toBeVisible();
+        await expect(dropdownContent.getByText(SECOND_TAB_TRIGGER_TEXT)).toBeDisabled();
+        await expect(dropdownContent.getByText(THIRD_TAB_TRIGGER_TEXT)).toBeVisible();
+
+        await dropdownContent.getByText(THIRD_TAB_TRIGGER_TEXT).click();
+
+        await page.waitForTimeout(500);
```

Comment:
Is it necessary now that you have the reduce motion?

if so still prefer a less "fixed" approach and use `waitForSelector`

Reply:
playwright is super flaky with these transitions and animations, there are several issues open about it.

With the waitFor i can get the test to pass but it isn't stable (~1 out of 5 tries just fails and it reads the wrong position) 
I really wanna avoid having a test in here thats not 100% stable